src/app/vars/animation.css

```diff
@@ -0,0 +1,16 @@
+:root {
+  --animation-fade-in: e-fade-in 500ms var(--easing-standard);
+  --animation-spin: e-rotate 500ms linear infinite;
+}
+
+@keyframes e-rotate {
```

Comment:
What's the `e` prefix stand for?

Reply:
it stands for e-oops - copy/paste fail from my stylguide 😄 - will fix